www/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "description": "React Bootstrap Documentation",
+  "version": "1.0.0",
+  "author": "Jason Quense",
+  "bugs": {
+    "url": "https://github.com/gatsbyjs/gatsby-starter-blog/issues"
+  },
+  "dependencies": {
+    "bootstrap": "^3.3.6",
+    "classnames": "^2.2.5",
+    "codemirror": "^5.16.0",
+    "gatsby": "^1.9.52",
+    "gatsby-link": "^1.6.18",
+    "gatsby-plugin-catch-links": "^1.0.8",
+    "gatsby-plugin-less": "^1.0.5",
+    "gatsby-plugin-offline": "^1.0.9",
+    "gatsby-remark-prismjs": "^1.2.8",
+    "gatsby-source-filesystem": "^1.5.1",
+    "gatsby-transformer-react-docgen": "^1.0.8",
+    "gatsby-transformer-remark": "^1.7.15",
+    "graphql": "^0.10.5",
+    "graphql-type-json": "^0.1.4",
+    "lodash": "^4.15.0",
+    "prop-types": "^15.6.0",
+    "react": "^15.5.4",
+    "react-bootstrap": "file:../",
```

Comment:
is this safe?

Reply:
should be haven't run into issues with it yet tho